app/notification_push.go

```diff
@@ -316,7 +316,7 @@ func DoesNotifyPropsAllowPushNotification(user *model.User, channelNotifyProps m
 	channelNotify, ok := channelNotifyProps[model.PUSH_NOTIFY_PROP]
```

Comment:
We aren't using the `ok` here. Perhaps we should be checking it, or otherwise disregarding it?

Reply:
Are are checking it in https://github.com/mattermost/mattermost-server/blob/03971a774c72abfd3b9516350fbe33e89e5ec1dc/app/notification_push.go#L337 and https://github.com/mattermost/mattermost-server/blob/03971a774c72abfd3b9516350fbe33e89e5ec1dc/app/notification_push.go#L347. I think this is fine, but 0/5.